src/about/team/TeamPage.vue

```diff
@@ -0,0 +1,56 @@
+<script setup lang="ts">
+import { VTLink } from '@vue/theme'
+import membersCoreData from './members-core.json'
+import membersEmeritiData from './members-emeriti.json'
+import membersPartnerData from './members-partner.json'
+import TeamHero from './TeamHero.vue'
+import TeamList from './TeamList.vue'
+
+const membersCore = membersCoreData
+const membersEmeriti = membersEmeritiData
+const membersPartner = membersPartnerData
+</script>
+
+<template>
+  <div class="TeamPage">
+    <TeamHero>
+      <template #title>Meet the Team</template>
+      <template #lead>The development of Vue and its ecosystem is guided by an international team, some of whom have chosen to be <span class="nowrap">featured below.</span></template>
+
+      <template #action>
+        <VTLink href="https://github.com/vuejs/governance/blob/master/Team-Charter.md">Learn more about teams</VTLink>
+      </template>
+    </TeamHero>
+
+    <TeamList :members="membersCore">
+      <template #title>Core Team Members</template>
+      <template #lead>In general, core team members are expected to maintain a consistent presence in the project. e also look for contributions over a longer period of time, so that we know the community can depend on the members long term.</template>
```

Comment:
Thanks a lot! Actually I wanted to discuss this text a bit. This text is taken from Vue Team Charter. However, maybe there's a better thing to say here. Like describes about what "core members" mean in this context. Any idea on re phrasing this entire text?

Maybe it should be like

> Core team members maintain a consistent presence in the project over a long period of time so that the community can depend on the members long term.

But I think there're better things say maybe 🤔  If we can't come up with any in the short term. We can just leave it as is (with the fix of @skirtles-code) for now 👍 

/cc @bencodezen